jumpscale/sals/reservation_chatflow/reservation_chatflow.py

```diff
@@ -389,14 +389,22 @@ def save_reservation(self, rid, name, solution_type, form_info=None):
         reservation.explorer = self._explorer.url
         reservation.save()
 
-    def get_solutions_explorer(self):
-        """delete old instances, to get the new ones from explorer
-        """
-        for obj in self.solutions.list_all():
-            self.solutions.delete(obj)
+    def get_solutions_explorer(self, deployed=True):
```

Comment:
when calling this by the other solutions, not the all solutions view, it will be called with the default deployed=True and will return all solutions right? but they wont be updated locally

Reply:
no, for update locally should call `update_local_reservations` not get, and by default `get_solution_explorer` return only deployed not all